plugin/trino-lance/src/main/java/io/trino/plugin/lance/catalog/namespace/DirectoryNamespaceConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.lance.catalog.namespace;
+
+import io.airlift.configuration.Config;
+
+public class DirectoryNamespaceConfig
```

Comment:
Please merge DirectoryNamespaceConfig and NamespaceTypeConfig into one. The new name should be LanceConfig. Also, add a TestLanceConfig class.

Reply:
@jackye1995 suggested we create individual config for different namespace(aka catalog) implementation: https://github.com/luohao/trino/pull/1#discussion_r2305010736

And it looks like other connectors also split the config for different catalog.